tests/Microsoft.Identity.Test.Integration.netcore/SeleniumTests/InteractiveFlowTests.NetFwk.cs

```diff
@@ -397,122 +397,5 @@ private SeleniumWebUI CreateSeleniumCustomWebUIForDuende(string username, string
                 loginBtn?.Click();
             }, TestContext);
         }
-
-        #region Azure AD Kerberos Feature Tests
-        [IgnoreOnOneBranch]
```

Comment:
nit: You can also remove this attribute, so this method will not be registered as a test, and keep the test code just in case it's needed, since Kerberos feature is still in MSAL. Once the feature is removed, all tests can be removed at the same time.

Reply:
I'm ok with ignoring (e.g. `[Ignore]`) the test like @pmaytak stated and soft deprecating `[Obsolete("msg", false)]` the `WithKerberosTicketClaim` API. We have telemetry that shows that nobody is using it.

Reply:
I think that should be a separate PR since I'll have to delete all 8 references to that method if I mark it as obsolete.